Xcode/xcconfigs/Project.xcconfig

```diff
@@ -15,7 +15,7 @@ GCC_WARN_ABOUT_MISSING_FIELD_INITIALIZERS = NO
 GCC_WARN_SIGN_COMPARE = NO
 GCC_TREAT_WARNINGS_AS_ERRORS                       = YES
 
-LITECORE_VERSION_STRING                            = 2.6.1
+LITECORE_VERSION_STRING                            = 2.6.2
```

Comment:
i was under the impression, this is only used by iOS.

Reply:
Yeah... I don't know how that number is used.

Core has changed: any Cobalt build (including iOS) will now be different than it was for 2.6.1.  I should think that would require updating that number.

Am I mistaken?

Reply:
This string gets written into the Info.plist if LiteCore is built as a (iOS/Mac) framework. That's all. Not a bad idea to keep it updated, but it won't affect our build process.